moto/networkmanager/urls.py

```diff
@@ -14,4 +14,5 @@
     "{0}/global-networks/(?P<networkid>[^/.]+)$": NetworkManagerResponse.dispatch,
     "{0}/tags$": NetworkManagerResponse.dispatch,
     "{0}/tags/(?P<resourcearn>[^/.]+)$": NetworkManagerResponse.dispatch,
+    "{0}/tags/(?P<arn_prefix>[^/]+)/(?P<resource_id>[^/]+)$": NetworkManagerResponse.dispatch,
```

Comment:
I see, the slash in the ARN was messing with the URL. Overlooked this details.
Interesting the non-servermode works Thanks!